spec/fixtures/proofing/lexis_nexis/ddp/error_response.json

```
@@ -1,7 +1,7 @@
 {
-  "error_detail": "service_type",
-  "request_id":"1234-abcd",
-  "request_result":"fail_invalid_parameter",
-  "review_status":"REVIEW_STATUS",
-  "tmx_summary_reason_code": ["Identity_Negative_History"]
-}
+    "error_detail": "service_type",
+    "request_id":"1234-abcd",
+    "request_result":"fail_invalid_parameter",
+    "review_status":"REVIEW_STATUS",
+    "tmx_summary_reason_code": ["Identity_Negative_History"]
+  }
```

I don't think we need these changes? The indentation is incorrect.

```suggestion
  "error_detail": "service_type",
  "request_id":"1234-abcd",
  "request_result":"fail_invalid_parameter",
  "review_status":"REVIEW_STATUS",
  "tmx_summary_reason_code": ["Identity_Negative_History"]
}
```